doc/nrf/releases_and_maturity/migration/migration_guide_2.7.rst

```diff
@@ -81,7 +81,16 @@ The following changes are recommended for your application to work optimally aft
 Samples and applications
 ========================
 
-This section describes the changes related to samples and applications.
+* For applications using the build types:
+
+  * The :makevar:`CONF_FILE` used for :ref:`app_build_additions_build_types` is now deprecated and is being replaced with the :makevar:`FILE_SUFFIX` variable, inherited from Zephyr.
+    You can read more about it in :ref:`app_build_file_suffixes`, :ref:`cmake_options`, and the :ref:`related Zephyr documentation <zephyr:application-file-suffixes>`.
+
+    If your application is using the build types, it is recommended to update the :file:`sample.yaml` to use the new variable instead of :makevar:`CONF_FILE`.
```

Comment:
```suggestion
    If your application uses build types, it is recommended to update the :file:`sample.yaml` to use the new variable instead of :makevar:`CONF_FILE`.
```